package.json

```diff
@@ -74,31 +81,19 @@
     "@types/whatwg-url": "^8.0.0",
     "@types/which": "^1.3.2",
     "atom-jasmine3-test-runner": "^5.1.7",
-    "build-commit": "0.1.1",
-    "cross-env": "7.0.2",
     "eslint": "^7.12.1",
     "eslint-config-atomic": "^1.5.1",
     "npm-check-updates": "9.2.4",
     "prettier": "^2.1.2",
-    "rollup": "2.33.1",
-    "rollup-plugin-atomic": "^1.2",
-    "shx": "0.3.3",
```

Comment:
Please keep the bundler. Bundling the code for the final release results in a much faster load time.

Reply:
I don't think we can use a bundler and transpiler since we can only have one `main` in package.json

I will try to run some tests and see what the difference is in speed between the two. If it is negligible (<~50ms) I think the speed up of development time is worth it.

Reply:
There is no need to have two entries. We can just run the release build in the end. This is similar to how C++ is compiled with Debug option during the development, but for releasing, it is compiled with Release option

Reply:
The release build will need `main` to point to `dist/terminal.js` but during development it will have to point to `src/terminal.ts`

I guess we could add changing this to the build script.

Reply:
[markdown-preview-plus](https://github.com/atom-community/markdown-preview-plus) also uses this. If there are limitations in terms of the entry file in ts-transpiler, we can make a PR to fix that.

Reply:
I see. markdown-preview-plus uses [loader.js](https://github.com/atom-community/markdown-preview-plus/blob/master/loader.js) to determine which to load.

I will try this out and see if I can get it to work.

Reply:
I created an issue: https://github.com/smhxx/atom-ts-transpiler/issues/72